packages/console/src/components/AppContent/index.tsx

```diff
@@ -19,7 +26,27 @@ const AppContent = ({ children, theme }: Props) => {
     };
   }, [theme]);
 
-  return <div className={styles.app}>{children}</div>;
+  useEffect(() => {
+    if (!isAuthenticated) {
+      void signIn(new URL(href, window.location.origin).toString());
+    }
+  }, [href, isAuthenticated, signIn]);
+
+  if (!isAuthenticated) {
+    return <>loading</>;
+  }
+
+  return (
+    <div className={styles.app}>
+      <Topbar />
+      <div className={styles.content}>
+        <Sidebar />
+        <Content>
```

Comment:
nit: Could remove the Content component here? using a `<div className="main"></div>` a little bit confuse with the upper className styles.content wrapper

Reply:
right